packages/playwright-core/src/server/registry/index.ts

```diff
@@ -1040,7 +1040,8 @@ export function buildPlaywrightCLICommand(sdkLanguage: string, parameters: strin
       return `pwsh bin/Debug/netX/playwright.ps1 ${parameters}`;
     default: {
       const packageManagerCommand = getPackageManagerExecCommand();
-      return `${packageManagerCommand} playwright ${parameters}`;
+      const binEntry = process.argv[1].endsWith(path.sep + 'playwright-core') ? 'playwright-core' : 'playwright';
```

Comment:
- There are more places that call `getPackageManagerExecCommand()`, and some of them need updating as well.
- Also, there are two versions of this function: `buildBasePlaywrightCLICommand` and `buildPlaywrightCLICommand` - let's merge them?
- IIUC, this relies on `argv` being something like `npx playwright-core install`? How does this work in the issue example, where I manually import from `playwright-core`?